msal4j-sdk/src/test/java/com/microsoft/aad/msal4j/ClaimsTest.java

```diff
@@ -30,30 +31,25 @@ void testClaimsRequest_Format() {
         cr.requestClaimInIdToken("sub", new RequestedClaimAdditionalInfo(true, "248289761001", null));
         cr.requestClaimInIdToken("auth_time", new RequestedClaimAdditionalInfo(false, null, null));
 
-        assertEquals(cr.formatAsJSONString(), TestConfiguration.CLAIMS_REQUEST);
+        assertEquals(TestConfiguration.CLAIMS_REQUEST, cr.formatAsJSONString());
     }
 
     @Test
     void testClaimsRequest_MergeWithClientCapabilitiesAndClaimsChallenge() throws URISyntaxException {
 
-        List<String> values = new ArrayList<>();
-        values.add("urn:mace:incommon:iap:silver");
-        values.add("urn:mace:incommon:iap:bronze");
-
         ClaimsRequest cr = new ClaimsRequest();
-        cr.requestClaimInAccessToken("given_name", new RequestedClaimAdditionalInfo(true, null, null));
-        cr.requestClaimInAccessToken("email", null);
-        cr.requestClaimInIdToken("acr", new RequestedClaimAdditionalInfo(false, null, values));
-        cr.requestClaimInIdToken("sub", new RequestedClaimAdditionalInfo(true, "248289761001", null));
-        cr.requestClaimInIdToken("auth_time", new RequestedClaimAdditionalInfo(false, null, null));
+        cr.requestClaimInAccessToken("nbf", new RequestedClaimAdditionalInfo(true, "1701477303", null));
+
+        Set<String> capabilities = new HashSet<>();
+        capabilities.add("cp1");
 
         PublicClientApplication pca = PublicClientApplication.builder(
```

Comment:
Prefer to write tests for confidential client. Public client should go through broker.

Reply:
This was an existing test that was adjusted to show claims behave correctly, but going forward I'll avoid adjusting old public client tests and continue to avoid adding new ones.